src/Control/Carrier/Cull/Church.hs

```diff
@@ -28,15 +32,24 @@ import Control.Monad.Trans.Class
 -- | Run a 'Cull' effect with the supplied continuations for '<|>', 'pure', and 'empty'. Branches outside of any 'cull' block will not be pruned.
 --
 --   prop> run (runCull (liftA2 (<|>)) (pure . pure) (pure empty) (pure a <|> pure b)) === [a, b]
+--
+-- @since 1.0.0.0
 runCull :: (m b -> m b -> m b) -> (a -> m b) -> m b -> CullC m a -> m b
 runCull fork leaf nil = runNonDet fork leaf nil . runReader False . runCullC
 
+-- | Run a 'Cull' effect, interpreting the result into an 'Alternative' functor. Choice is handled with '<|>', embedding with 'pure', and failure with 'Control.Applicative.empty'.
```

Comment:
```suggestion
-- | Run a 'Cull' effect, interpreting the result into an 'Alternative' functor. Choice is handled with '<|>', embedding with 'pure', and failure with 'empty'.
```

Reply:
Tho actually, I’m not sure we need to say all of this; “mapping the result into an `Alternative` functor” would seem to suffice.